patterns/hero.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Title: Hero
+ * Slug: twentytwentyfour/hero
+ * Categories: banner
+ */
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"6vh","bottom":"6vh"}}},"layout":{"type":"constrained"}} -->
```

Comment:
Instead of arbitrary values, let's use the spacing presets for paddings throughout.